### ISSUE TITLE
Kernel error - restarting. When .sgt with more than 288 data

### ISSUE THREAD

**Issue by Pbottelin** [gimli-org/gimli#222]
Hi, I face a strange error with a refraction traveltime inversion since yesterday.

Environment:
Windows, pygimli 1.0, Anaconda distribution

I have a datafile PS03.sgt containing 600 measurements (49 shot/sensors). It causes the python kernel to fail/restart (in spyder).

When I delete some data (about half the shots, i.e. <290 measurements, PS04.sgt file), the inversion goes fine.

I tried processing each shot individually to check their validity: no problem, the raypath appear consistent.

I attach the files:
PS03: full data -> FAILS
PS04: half the shots -> WORKS Fine

[MWE_pygimli_tomography_kernel_dead.zip](https://github.com/gimli-org/gimli/files/4255984/MWE_pygimli_tomography_kernel_dead.zip)

Do you have any idea what's going on?

Cheers,
best regards

Pierre

**Comment by halbmy** [comment]
Well, the code runs nicely with both PS03 and PS04 on my Windows pg 1.0.12 version. Here's the result for PS03 (a bit rougher than PS04):

![image](https://user-images.githubusercontent.com/7136401/75374665-f9752c80-58cc-11ea-9ac0-52a437be66c7.png)

**Comment by Pbottelin** [comment]
Hi Thomas,
I re-tried with both files, and I still get the same error.
The output of pg.__version__ yields '0+untagged.2392.gbc43dbc)'
I think it is pg 1.0.11 on windows.

I discussed with my colleague Aurore Carrier, she faced the same issue with another file. She thinks it might originate from conflict between source/sensors positions (as it works when we suppress some data). For her issue, the inversion worked after suppressing 1 shot.
Maybe we did something wrong in the meshing or with the .sgt itself? Positions x,y might contain too much decimal numbers?

Anyway, I downloaded the pg 1.012 version (whl file from the website) and installed it with pip install. It runs nicely now!
I'll check if it solves also Aurore's issue.

Thanks for the version clue,
best regards

Pierre
